python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -159,6 +159,8 @@ class ScheduleEvaluationContext:
             from both the actual execution time and the time at which the run config is computed.
             Not available in all schedulers - currently only set in deployments using
             DagsterDaemonScheduler.
+        resources (Optional[Dict[str, Any]]): Mapping of resource name to resource
```

Comment:
Do we use "resource name" elsewhere? I think this should probably be "resource key"?